src/PSRule.Rules.Azure/PSRule.Rules.Azure.psd1

```diff
@@ -55,7 +55,6 @@ RequiredModules = @(
     @{ ModuleName = 'PSRule'; ModuleVersion = '0.0.1' }
     @{ ModuleName = 'Az.Accounts'; ModuleVersion = '1.5.2' }
```

Comment:
@ArmaanMcleod Lets bump this `Az.Accounts` version. `Invoke-AzRestMethod` was added in v1.9.1 to at least this version. We also use v2.5.2 in the build pipeline so that would be ok too.

Reply:
@BernieWhite Good point. I've bumped the version to 2.5.2 🙂